tools/elk/README.md

```diff
@@ -0,0 +1,54 @@
+# Analyzing Marathon with ELK Stack
+
+Marathon comes with a basic set of grok filter for Logstash to load logs into
+Elasticsearch. This should enable users to analyze logs with Kibana.
+
+## Generating LogStash config
```

Comment:
it's Logstash not LogStash - just a cosmetic... :)

Reply:
but if you agree, please fix it also on other places where that appears

Reply:
thanks for pointing this out! Fixed.